client/js/map.js

```diff
@@ -378,6 +378,20 @@ function getClusterRadius(zoom) {
     }
 }
 
+function shortenLargeNumber(number, digits) {
+    var units = ['K', 'M', 'G', 'T', 'P', 'E', 'Z', 'Y'],
+        decimal;
+
+    for (var i = units.length - 1; i >= 0; i--) {
+        decimal = Math.pow(1000, i + 1);
+
+        if (number <= -decimal || number >= decimal) {
+            return +(number / decimal).toFixed(digits) + units[i];
+        }
```

Comment:
Now that we're not using any decimal places, we should be able to remove the logic pertaining to the `decimal` variable too.

Reply:
Whirish, the decimal variable is used to define which level will be used so it can't be removed, then I removed the "digits" parameter because we're not going to use decimal places, I keep the logic right.

Reply:
You're absolutely correct, my bad